api/cosoul/[address].ts

```diff
@@ -75,7 +91,9 @@ async function getCosoulData(address: string) {
             },
             // what is the diff between pgive and normalized_pgive.
             // I thought pgive was normalized give, plus stuff
```

Comment:
@wingfeatherwave @CryptoGraffe have we figured out this difference yet and can we remove this comment?